sample/src/main/java/ch/sonect/sdk/shop/integrationapp/MainActivity.kt

```diff
@@ -1,15 +1,22 @@
 package ch.sonect.sdk.shop.integrationapp
 
+import android.content.ClipData
+import android.content.ClipboardManager
+import android.content.Context
 import android.os.Bundle
 import android.util.Base64
+import android.widget.Toast
 import androidx.appcompat.app.AppCompatActivity
+import ch.sonect.common.extension.afterTextChanged
 import ch.sonect.sdk.shop.SonectSDK
 import kotlinx.android.synthetic.main.activity_main.*
 import javax.crypto.Mac
 import javax.crypto.spec.SecretKeySpec
 
 class MainActivity : AppCompatActivity() {
 
+    lateinit var sharedPreferences: CacheManager
```

Comment:
`private val sharedPreferences by lazy { CacheManager(this) } `

Reply:
done